unknown_type/[M]676.cpp

```diff
@@ -0,0 +1,40 @@
+// Problem: 676. Implement Magic Dictionary - Medium
+// Link: https://leetcode.com/problems/implement-magic-dictionary/
+
+/* Nhận xét:
+- I have nothing to say about this :(
+*/
+
+class MagicDictionary {
+private:
+    vector<vector<string>> dict;
```

Comment:
Ok. Code thế này cũng tạm được rồi :))

Reply:
Cách "rất được" thì lại là dùng trie ah? :))

Reply:
Trie bài này cũng loằng ngoằng. Cơ mà lúc pv a cũng ko nghĩ nó cho bài này để ứng viên nghĩ ra Trie đâu. Chú code thuật toán tuy ko tối ưu nhất nhưng sạch đẹp thì vẫn có offer thôi .